docs/release_notes/v0.9.1-alpha.md

```diff
@@ -0,0 +1,30 @@
+The v0.9.1 release adds `flightsql` data connector and `sqlite` data store. Improved the usability by adding more `spice` CLI commands.
+
+## Highlights in v0.9.1-alpha
+
+**FlightSQL data connector**: [Arrow Flight SQL](https://arrow.apache.org/docs/format/FlightSql.html) can now be used as a connector for federated SQL query.
+
+**SQlite data backend**: [SQLite](https://www.sqlite.org/) can now be used as a data store for acceleration.
+
+## Contributors
+
+- @phillipleblanc
+- @mitchdevenport
+- @Jeadie
+- @ewgenius
+- @sgrebnov
+- @y-f-u
+- @lukekim
+
+## New in this release
+
+- **Adds** FlightSQL data connector (`flightsql`).
+- **Adds** SQlite data store.
+- **Adds** support for date, varchar, bpchar, primitive list type in Postgres.
+- **Adds** `spice pods`, `spice status`, `spice datasets`, and `spice models` CLI commands.
+- **Adds** `v1/spicepods` API.
+- **Adds** `spiced` docker build and release.
+- **Adds** e2e tests for release installation and local acceleration.
+- **Fixes** `spice sql` REPL panics when query result is too large.
```

Comment:
@y-f-u Please add the link to the Issue for bug fixes.

Reply:
Also, maybe just say `bug fixes for spice sql` and then list them